### PR TITLE
Initial views for a monitoring dashboard

### PR DESCRIPTION
**Pull request by aptemus**
Screenshot of dashboard available on Jira ticket.
This lets us monitor activity in NTBS:
- notifications created & on demand data migrations done
- alerts created and closed
- how the Hangfire jobs did last night/last time
- user logins, user edits, record reads
- specimens unmatched

### CODE REVIEW

File: source/dbo/Views/Power BI Reporting/vwHangfireJobs.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿CREATE VIEW [dbo].[vwHangfireJobs]
+	AS
+	SELECT 
+		SUBSTRING(Q1.InvocationData, 25, EndOfString-25) AS JobName, 
+		Q1.StateName, 
+		Q1.CreatedAt 
+	FROM
+		(SELECT StateName, CreatedAt, CHARINDEX(',',InvocationData, 1) AS EndOfString, InvocationData
+		FROM [$(NTBS)].[HangFire].[Job]) AS Q1
```
Comment (benjimarshall) [inline source/dbo/Views/Power BI Reporting/vwHangfireJobs.sql:9]: I don't think this is going to play well with the on-demand migration records in Hangfire. An example of one is 

```
{"t":"ntbs_service.DataMigration.INotificationImportService, ntbs-service","m":"BulkImportByDateAsync","p":["Hangfire.Server.PerformContext, Hangfire.Core","System.Int32","System.DateTime, mscorlib","System.DateTime, mscorlib"]}
```
And an example row this produces is:
| JobName | StateName | CreatedAt |
| ---- | --------- | ------ |
| `igration.INotificationImportService` | `Deleted` | `2021-06-15 11:06:27.920` |

which I don't think was the intended result

**Review by benjimarshall — CHANGES_REQUESTED** [review]
Just one problem I think I've spotted but otherwise it looks good!

File: source/dbo/Views/Power BI Reporting/vwHangfireJobs.sql (hunk 1 above)
Comment (aptemus) [reply source/dbo/Views/Power BI Reporting/vwHangfireJobs.sql:9]: @benjimarshall Yeah I'll add a WHERE clause to only look for records where the Invocation data includes 'ntbs_service.Jobs.'

**Review by benjimarshall — APPROVED** [review]
Looks good!

**Pull request merged**